tests/integration/similarity/test_integration_similarity_notebooks.py

```diff
@@ -26,3 +25,17 @@ def test_01_notebook_run(similarity_notebooks):
 
         nb_output = sb.read_notebook(OUTPUT_NOTEBOOK)
         assert nb_output.scraps["median_rank"].data <= 10
+
+
+@pytest.mark.notebooks
+def test_11_notebook_run(similarity_notebooks, tiny_ic_data_path):
+    if linux_with_gpu():
+        notebook_path = similarity_notebooks["11"]
+        pm.execute_notebook(
+            notebook_path,
+            OUTPUT_NOTEBOOK,
+            parameters=dict(PM_VERSION=pm.__version__),
+            kernel_name=KERNEL_NAME,
+        )
+        # nb_output = sb.read_notebook(OUTPUT_NOTEBOOK)
```

Comment:
remove commented lines (if all we're testing for is that it runs)

Reply:
Done